src/main/java/org/jenkinsci/test/acceptance/plugins/gradle/GradleTask.java

```diff
@@ -0,0 +1,30 @@
+package org.jenkinsci.test.acceptance.plugins.gradle;
+
+/**
+ * Created by wlaschinger on 29.06.17.
+ */
+public enum  GradleTask {
+
+    HELLO("hello", "Hello world!"),
+    FIRST("firstTask", "First!"),
+    SECOND("secondTask", "Second!"),
+    ENVIRONMENT_VARIABLES("environmentVariables", null),
```

Comment:
Using null as a parameter is a code smell: better introduce additional one-arg constructor.